### ISSUE TITLE
Add further steps to Development ReadMe

### ISSUE THREAD

**Issue by lewinpauli** [oae/gnome-shell-pano#281]
I had no Idea what I should do after building the project by myself to use this awesome extension in gnome 46

like mentioned by "l0sty" on the gnome extension page  (https://extensions.gnome.org/extension/5278/pano/),
it would be useful to have a documentation to the further steps
 
 As far as I know you need to Logout from your gnome session
 
 and after that running "gnome-extensions enable pano@elhan.io"

**Comment by Totto16** [comment]
We certainly could add a better description on how to build the extension yourself and then how to install it, but our main goal is to just publish it on EGO and then it's just a one click install.

It is only due to the third party dependencies and the delay of #278 that the version supporting gnome 46 isn't on EGO yet. Partly also due to my lack of time 😓

**Comment by lewinpauli** [comment]
@Totto16 I can understand that the preferred way is for the user to use the official EGO way, but since it takes quite some time to support gnome 46 and newer, I think just another 3 (beginner friendly) lines like this would help a lot to build the clipboard for non-EGO supported versions:

At the beginning of the build documentation:

'If you want to debug/develop the extension, or if you need to build the extension yourself because it is not currently in the GNOME extension store for your version, you can build it yourself:'


At the end of the build documentation:

'Log out of your current GNOME session (normal logout, next to reboot, etc.) 

Activate the pano extension in your "GNOME Extensions" application.'

**Comment by Totto16** [comment]
> [@Totto16](https://github.com/Totto16) I can understand that the preferred way is for the user to use the official EGO way, but since it takes quite some time to support gnome 46 and newer, I think just another 3 (beginner friendly) lines like this would help a lot to build the clipboard for non-EGO supported versions:
> 
> At the beginning of the build documentation:
> 
> 'If you want to debug/develop the extension, or if you need to build the extension yourself because it is not currently in the GNOME extension store for your version, you can build it yourself:'
> 
> At the end of the build documentation:
> 
> 'Log out of your current GNOME session (normal logout, next to reboot, etc.)
> 
> Activate the pano extension in your "GNOME Extensions" application.'

You are right. I'll write one (soonish ™ ). But you are also welcome to make a PR that does that. :)
